### PR TITLE
normal distribution from bedpe data 

### PR DESCRIPTION
**Pull request by cfauci**
# Description
this is still a work in progress, but as of now functions properly (besides the cmd/bedMath/bedMath.go). this is designed to calculate a normal distribution from a bedpe for the curve that describes the number of contacts as a function of distance from any bin in the genome.

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [ ] I performed a self-review of my code
- [ ] If it this a core feature, I have added thorough tests
- [ ] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues
Need to add capability of - strand to functionality

### CODE REVIEW

File: numbers/curveFit/curveFit.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package curveFit
```
Comment (cfauci) [inline numbers/curveFit/curveFit.go:1]: this file will probably be deleted but I'm still working on how to do the sampling and fitting that I need to do
